.github/CODEOWNERS

```diff
@@ -1,2 +1,30 @@
 # Default code owners for entire repository
 *   @SimiHunjan @ochikov @petreze @svetoslav-nikol0v @agadzhalov
+
+#########################
+#####  Core Files  ######
+#########################
+
+# NOTE: Must be placed last to ensure enforcement over all other rules
+
+# Protection Rules for Github Configuration Files and Actions Workflows
+/.github/                                       @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/.github/workflows/                             @hashgraph/release-engineering @hashgraph/release-engineering-managers @SimiHunjan @ochikov @petreze @svetoslav-nikol0v @agadzhalov
```

Comment:
**Suggestion**: to create a group for the maintainers as in [C++](https://github.com/hashgraph/hedera-sdk-cpp/blob/main/.github/CODEOWNERS)